mysql-test/main/perror-win.result

```diff
@@ -2,7 +2,7 @@ MariaDB error code 150: Foreign key constraint is incorrectly formed
 Win32 error code 150: System trace information was not specified in your CONFIG.SYS file, or tracing is disallowed.
 OS error code  23:  Too many open files in system
 Win32 error code 23: Data error (cyclic redundancy check).
-MariaDB error code 1062 (ER_DUP_ENTRY): Duplicate entry '%-.192T' for key %d
+MariaDB error code 1062 (ER_DUP_ENTRY): Duplicate entry '%-.192sT' for key %d
```

Comment:
Hollup. How did so many specifiers end up in `.result`s verbatim? Are they intended?

Reply:
They’re mostly likely are intended.